multipart-form-data.opam

```diff
@@ -6,18 +6,18 @@ bug-reports: "https://github.com/cryptosense/multipart-form-data/issues"
 license: "BSD-2"
 dev-repo: "https://github.com/cryptosense/multipart-form-data.git"
 build: [
-  [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" ]
+  ["jbuilder" "subst" "-n" name] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
 ]
 build-test: [
-  [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "true" ]
-  [ "ocaml" "pkg/pkg.ml" "test" ]
+  ["jbuilder" "runtest" "-p" name "-j" jobs]
 ]
 depends: [
-  "topkg" {build}
-  "ocamlfind" {build}
+  "topkg-jbuilder" {build}
```

Comment:
`topkg-jbuilder` is only necessary for making releases FYI. You should list `jbuilder` as your build dependency instead.

Reply:
Ah, thanks, I was not sure about that. Is `ocaml-migrate-parsetree` as `{build}` OK? I'm using it through jbuilder to process `lwt.ppx`.

Reply:
If your ppx uses omp, then it will pull it already. That being said, you're right that depending on transitive is probably wrong. I'd put it...

Reply:
ok, thanks!